zfmt/ast.go

```diff
@@ -621,6 +637,10 @@ func (c *canon) fromEntity(e ast.FromEntity) {
 		c.pattern(e)
 	case *ast.Name:
 		c.write(zson.QuotedName(e.Text))
+	case *ast.CrossJoin:
+		c.write("cross join XXX")
+	case *ast.SQLJoin:
+		c.write("sql join XXX")
```

Comment:
Are you deliberately leaving these unfinished?

Reply:
yes

Reply:
I could take them out and get the panic.  -C for sql is a whole little project.  e.g., select isn't handled yet at all.

Reply:
I'd go with panic but this is fine.